restler/restler_settings.py

```diff
@@ -909,10 +964,12 @@ def validate_options(self):
         if self.fuzzing_mode == 'random-walk' and self.max_sequence_length != 100:
             raise OptionValidationError("Should not provide maximum sequence length"
                                         " for random walk method")
-        if self.token_refresh_interval and not self.token_refresh_cmd:
-            raise OptionValidationError("Must specify command to refresh token")
-        if self.token_refresh_cmd and not self.token_refresh_interval:
+        if self.token_refresh_interval and not self.token_authentication_method:
```

Comment:
>if self.token [](http://example.com/codeflow?start=8&length=13)

Suggest factoring this out to a function 'validate_auth_options()' to separate these out (perhaps a local function)?

Reply:
sure, adding a local function validate_auth_options()